[New Commers Issue] script deprecation warning

While building a miniRADI, there is a warning in an old script that we use that leads to 1 min wait time, replace this script with an ideal updated one.

`RUN curl -sL https://deb.nodesource.com/setup_16.x | bash -   && apt-get install -y nodejs   && npm install -g yarn `

![image](https://github.com/JdeRobot/RoboticsAcademy/assets/40732709/5db3f817-3a16-483a-b51f-9442ccfa2dd2)

It failed on my system. Don't know whether it is because of my change or some other problem. Here, is the error.
```bash
=> ERROR [10/26] RUN /bin/bash -c '. /opt/ros/$ROS_DISTRO/setup.bash; cd /catkin_ws; catkin build'                                                                                                  540.6s 
------                                                                                                                                                                                                      
 > [10/26] RUN /bin/bash -c '. /opt/ros/$ROS_DISTRO/setup.bash; cd /catkin_ws; catkin build':                                                                                                               
3.108 -----------------------------------------------                                                                                                                                                       
3.108 Profile:                     default                                                                                                                                                                  
3.108 Extending:             [env] /opt/ros/noetic                                                                                                                                                          
3.108 Workspace:                   /catkin_ws                                                                                                                                                               
3.108 -----------------------------------------------
3.108 Build Space:        [exists] /catkin_ws/build
3.108 Devel Space:        [exists] /catkin_ws/devel
3.108 Install Space:      [unused] /catkin_ws/install
3.108 Log Space:         [missing] /catkin_ws/logs
3.108 Source Space:       [exists] /catkin_ws/src
3.108 DESTDIR:            [unused] None
3.108 -----------------------------------------------
3.108 Devel Space Layout:          linked
3.108 Install Space Layout:        None
3.108 -----------------------------------------------
3.108 Additional CMake Args:       None
3.108 Additional Make Args:        None
3.108 Additional catkin Make Args: None
3.108 Internal Make Job Server:    True
3.108 Cache Job Environments:      False
3.108 -----------------------------------------------
3.108 Buildlisted Packages:        None
3.108 Skiplisted Packages:         None
3.108 -----------------------------------------------
3.108 Workspace configuration appears valid.
3.108 
3.108 NOTE: Forcing CMake to run for each package.
3.108 -----------------------------------------------
3.108 [build] Found 24 packages in 0.0 seconds.                                      
3.108 [build] Updating package table.                                                
3.108 Starting >>> catkin_tools_prebuild                                             
Finished <<< catkin_tools_prebuild                  [ 2.5 seconds ]            ld... 
5.621 Starting >>> drone_assets                                                      
5.621 Starting >>> drone_circuit_assets                                              
5.621 Starting >>> kobuki_gazebo_plugins                                             
5.621 Starting >>> mavlink                                                           
5.621 Starting >>> mavros_msgs                                                       
5.621 Starting >>> rqt_ground_robot_teleop                                           
5.621 Starting >>> stdr_msgs                                                         
5.621 Starting >>> stdr_resources                                                    
Finished <<< stdr_resources                         [ 4.0 seconds ]             0.1] 
Finished <<< drone_circuit_assets                   [ 4.1 seconds ]            4.... 
Finished <<< rqt_ground_robot_teleop                [ 5.2 seconds ]            in... 
Finished <<< mavlink                                [ 5.7 seconds ]            in... 
Finished <<< stdr_msgs                              [ 15.5 seconds ]           in... 
21.04 Starting >>> libmavconn                                                        
Finished <<< mavros_msgs                            [ 1 minute and 44.3 seconds ]... 
Finished <<< libmavconn                             [ 5 minutes and 54.9 seconds ].. 
Failed <<< drone_assets                           [ 6 minutes and 34.5 seconds ]e... 
400.6 Abandoned <<< mavros                                 [ Unrelated job failed ]  
400.6 Abandoned <<< stdr_parser                            [ Unrelated job failed ]  
400.6 Abandoned <<< stdr_samples                           [ Unrelated job failed ]  
400.6 Abandoned <<< stdr_server                            [ Unrelated job failed ]  
400.6 Abandoned <<< jderobot_drones                        [ Unrelated job failed ]  
400.6 Abandoned <<< drone_wrapper                          [ Unrelated job failed ]  
400.6 Abandoned <<< mavros_extras                          [ Unrelated job failed ]  
400.6 Abandoned <<< rotors_driver                          [ Unrelated job failed ]  
400.6 Abandoned <<< rqt_drone_teleop                       [ Unrelated job failed ]  
400.6 Abandoned <<< stdr_launchers                         [ Unrelated job failed ]  
400.6 Abandoned <<< stdr_robot                             [ Unrelated job failed ]  
400.6 Abandoned <<< stdr_gui                               [ Unrelated job failed ]  
400.6 Abandoned <<< stdr_simulator                         [ Unrelated job failed ]  
400.6 Abandoned <<< tello_driver                           [ Unrelated job failed ]  
400.6 Abandoned <<< test_mavros                            [ Unrelated job failed ]  
Finished <<< kobuki_gazebo_plugins                  [ 6 minutes and 34.9 seconds ].. 
401.0 [build] Summary: 9 of 25 packages succeeded.                                   
401.0 [build] Ignored: None.                                                         
401.0 [build] Warnings: None.                                                        
401.0 [build] Abandoned: 15 packages were abandoned.                                 
401.0 [build] Failed: 1 packages failed.                                             
401.0 [build] Runtime: 6 minutes and 37.9 seconds total.                             
401.0 [build] Note: Workspace packages have changed, please re-source setup files to use them.
401.0 _______________________________________________________________________________
401.0 Errors << drone_assets:make /catkin_ws/logs/drone_assets/build.make.000.log    
401.0 c++: fatal error: Killed signal terminated program cc1plus
401.0 compilation terminated.
401.0 make[2]: *** [CMakeFiles/wall1plugin.dir/build.make:63: CMakeFiles/wall1plugin.dir/plugins/wall1.cc.o] Error 1
401.0 make[1]: *** [CMakeFiles/Makefile2:268: CMakeFiles/wall1plugin.dir/all] Error 2
401.0 make[1]: *** Waiting for unfinished jobs....
401.0 c++: fatal error: Killed signal terminated program cc1plus
401.0 compilation terminated.
401.0 make[2]: *** [CMakeFiles/wall3plugin.dir/build.make:63: CMakeFiles/wall3plugin.dir/plugins/wall3.cc.o] Error 1
401.0 make[1]: *** [CMakeFiles/Makefile2:889: CMakeFiles/wall3plugin.dir/all] Error 2
401.0 c++: fatal error: Killed signal terminated program cc1plus
401.0 compilation terminated.
401.0 make[2]: *** [CMakeFiles/wall2plugin.dir/build.make:63: CMakeFiles/wall2plugin.dir/plugins/wall2.cc.o] Error 1
401.0 make[1]: *** [CMakeFiles/Makefile2:241: CMakeFiles/wall2plugin.dir/all] Error 2
401.0 make: *** [Makefile:141: all] Error 2
401.0 cd /catkin_ws/build/drone_assets; catkin build --get-env drone_assets | catkin env -si  /usr/bin/make --jobserver-auth=3,4; cd -
401.0 
401.0 ...............................................................................
401.0 Failed << drone_assets:make                       [ Exited with code 2 ]       
------
Dockerfile.mini_noetic:29
--------------------
  27 |     
  28 |     RUN cd /catkin_ws && rosdep update && rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO -y
  29 | >>> RUN /bin/bash -c '. /opt/ros/$ROS_DISTRO/setup.bash; cd /catkin_ws; catkin build'
  30 |     
  31 |     #ROScontrol and LIDAR
--------------------
ERROR: failed to solve: process "/bin/sh -c /bin/bash -c '. /opt/ros/$ROS_DISTRO/setup.bash; cd /catkin_ws; catkin build'" did not complete successfully: exit code: 1

```

--- a/scripts/mini_RADI/Dockerfile.dependencies_humble
+++ b/scripts/mini_RADI/Dockerfile.dependencies_humble
@@ -132,7 +132,7 @@ RUN apt-get update && \
 ENV PATH "$PATH:/opt/VirtualGL/bin:/opt/TurboVNC/bin"
 
 # Node
-RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - \
+RUN curl -sL https://deb.nodesource.com/setup_20.x | bash - \
   && apt-get install -y nodejs \
   && npm install -g yarn
 

